src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -184,4 +183,106 @@ public void Chat_007_banUser() {
     chatUserStaff.banUser(userToBeBanned);
     chatUserStaff.unBanUser(userToBeBanned);
   }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_008", "Chat", "Modals", "ChatTests"})
+  public void Chat_008_userBlockedCanNotCreatePrivateMessage() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)